javascript/ql/src/semmle/javascript/frameworks/Angular2.qll

```diff
@@ -453,7 +471,7 @@ module Angular2 {
 
     /** Gets the name of the variable holding the element of the current iteration. */
     string getIteratorName() {
-      result = getValue().regexpCapture("^ *let (\\w+) .*", 1)
+      result = getValue().regexpCapture(" *let +(\\w+).*", 1)
```

Comment:
Would be nice to see an example where the removed anchor is needed (a test case?).

Reply:
The anchor had no effect, so I removed it. (`regexpCapture` wants to match the whole string)